package-lock.json

```diff
@@ -45008,6 +45000,23 @@
         "node": ">=8"
       }
     },
+    "packages/compass-e2e-tests/node_modules/debug": {
+      "version": "4.3.4",
```

Comment:
Hmm, weird that this is getting added here if we aligned all the versions 🤔

Reply:
Nice catch, ran the manual hosting steps we do in `update-electron`. Looks like it got it this time.